news/2020-01-03-monthly-beatmapping-contests-return.md

```diff
@@ -0,0 +1,135 @@
+
+For transparency's sake, the screening users this month are [ScubDomino](https://osu.ppy.sh/users/8972308), [Noffy](https://osu.ppy.sh/users/1541323), [Zelq](https://osu.ppy.sh/users/8953955), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), [Smokeman](https://osu.ppy.sh/users/2140676), [Sing](https://osu.ppy.sh/users/3795679), [Cris-](https://osu.ppy.sh/users/6175280), [Realazy](https://osu.ppy.sh/users/918297), and [Nuvolina](https://osu.ppy.sh/users/10974170). In case any of them drop out, back-up screening users are [Myxo](https://osu.ppy.sh/users/2202645), [Cheri](https://osu.ppy.sh/users/5226970), and [Sinnoh](https://osu.ppy.sh/users/4236057). All of these users will be reviewing the submissions without knowledge of who their creators are, and each of their preferred submissions will be viewable when the top 5 entries are announced.
+
+## <a id="voting"></a>Voting
+
+Instead of relying solely on mappers for their input, this contest also relies on opinions from osu! players (like you!). 
+
+The final two weeks of each contest are reserved for community voting. Users can download and play through the top 5 entries and provide a vote for their favorite. Given each month's contest features a mapping limitation, it's recommended to prioritize votes for entries that most effectively implement the limitation!
+
+The voting page for this contest will be accessible upon the announcement of next month's contest.
+
+## <a id="rewards"></a>Rewards
+
+Similar to the rewards structure of the [Mappers' Guild](https://mappersguild.com), rewards for the Monthly Beatmapping Contest will be based on cumulative points earnings:
+- **Community vote winner:** 2 points
+- **Community vote runner-up:** 1 point
+- **Screening crew's #1 pick:** 1 point
+
+Each user's MBC points will be listed at the end of the news posts announcing a contest's winners.
+
+MBC points reward users at certain thresholds:
+- **Profile badge:** badges are attainable at 3, 6, and 10 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
+- **"Elite Mapper" forum title**: attainable at 10 points.
+- **osu!supporter:** each point earned provides a user with a month of supporter for themself or someone of their choosing.
+- **Mappers' Guild participation:** participants who are in the Mappers' Guild will receive small earnings within the guild for submitting a valid entry (5), screening (1), and voting (1). These values are unrelated to the contest's points.
+
+## Rules
+
+So how do you participate? Submissions must...
+
+- **...follow the month's [beatmapping limitation](#limitation): _all slider shapes must be straight lines (2 anchors)_.**
```

Comment:
i think ``(2 anchors)`` might sound quite misleading to some people (wrongly assuming they could use straight slider + 2 red anchors), might be better going with something like ``(no extra sliderpoints)`` or so i guess?